src/repl.jl

```diff
@@ -3,27 +3,34 @@
 # Then, at any time, press Ctrl-O to edit in normal mode.
 # To be able to execute code and search history etc, going back to insert mode is neccesary.
 # For the moment, the first keypress after going to insert mode disappears sometimes.
-import Base: LineEdit, REPL
-using Neovim
-import Neovim: get_cursor, set_cursor, command, input
+# import Base: LineEdit, REPL
+import REPL
+import REPL.LineEdit
 
-type NvimReplState
+include("Neovim.jl")
+
+# using Neovim
+# import Neovim: get_cursor, set_cursor, command, input
```

Comment:
Can remove these comments

Reply:
Fixed in https://github.com/bfredl/Neovim.jl/commit/97ae01b61b6696dcbf3ed8ccfd285658311cb256